pychromecast/__init__.py

```diff
@@ -122,13 +129,14 @@ class Chromecast(object):
     :param retry_wait: A floating point number specifying how many seconds to
                        wait between each retry. None means to use the default
                        which is 5 seconds.
+    :param dynamic_host: Dynamic or non_dynamic host, if True dynamic host
+                         tracking is enabled. Will be True by default for
+                         cast_type == CAST_TYPE_GROUP.
     """
 
     def __init__(self, host, port=None, device=None, **kwargs):
```

Comment:
Instead of storing `kwargs`, please write out all the attributes that we can have and their default values and store as instance variables.

```python
def __init__(self, host, port=None, device=None, *, tries=None, timeout=None, retry_wait=None, blocking=True):
    self._tries = tries
    self._timeout = timeout
```